src/Symfony/Bundle/FrameworkBundle/Command/ConfigDebugCommand.php

```diff
@@ -190,36 +185,26 @@ private function getConfigForExtension(ExtensionInterface $extension, ContainerB
     public function complete(CompletionInput $input, CompletionSuggestions $suggestions): void
     {
         if ($input->mustSuggestArgumentValuesFor('name')) {
-            $suggestions->suggestValues($this->getAvailableBundles());
+            $suggestions->suggestValues($this->getAvailableBundles(!preg_match('/[A-Z]/', $input->getCompletionValue()[0] ?? '')));
```

Comment:
That avoids suggesting twice the each bundle with syntax (`FrameworkBundle` and `framework`) and switch to the must accurate suggestions when something has been typed.

```suggestion
            $suggestions->suggestValues($this->getAvailableBundles(!preg_match('/^[A-Z]/', $input->getCompletionValue())));
```

Reply:
I replaced my code by your suggestion @GromNaN.